packages/cli/src/playground/vite.config.ts

```diff
@@ -27,13 +27,17 @@ export default defineConfig(({ mode }) => {
   };
 
   if (mode === 'development') {
+    // Use environment variable for the target port, fallback to 4111
+    const targetPort = process.env.MASTRA_DEV_PORT || '4111';
```

Comment:
This is fine for monorepo testing I guess, but it's not used when using mastra dev

Reply:
yeah, this is mainly just for developing in the playground if we use a diff port